exercise_29.js

```diff
@@ -0,0 +1,18 @@
+/*
+Implement the getFahrenheitConvertedToCelcius function
+
+it should accept one argument:
+
+a number
+
+the function should return the number converted from Fahrenheit to Celcius
+ */
+
+
+const celciusDegree = 33.8
+function getFahrenheitConvertedToCelcius (number) {
+    return number / celciusDegree;
```

Comment:
to NIE działa dobrze bo `100 °F  =  37.7778 °C` a u ciebie to `100 °F  =  2.958579881656805 °C`